### PR TITLE
Fix a bug with the urls for the help system

### PR DESCRIPTION
**Pull request by cquiroz**
Also updates `github-markdown-css`

### CODE REVIEW

File: explore/src/main/scala/explore/HelpBody.scala [hunk 1]
```diff
@@ -75,7 +76,7 @@ object HelpBody {
         load(props.url).flatMap(v => state.set(Pot.fromTry(v)).to[IO])
       }
       .render { (props, state) =>
-        val imageConv = (s: Uri) => props.baseUrl.withPath(s.path)
+        val imageConv = (s: Uri) => props.baseUrl.withPath(props.baseUrl.path.concat(s.path))
```
Comment (toddburnside) [inline explore/src/main/scala/explore/HelpBody.scala:79]: Seems weird that you need to do it this way. But, if it works, it works. :)

**Review by toddburnside — APPROVED** [review]
LGTM

File: explore/src/main/scala/explore/HelpBody.scala [hunk 2]
```diff
@@ -36,7 +36,8 @@ final case class HelpBody(base: HelpContext, helpId: Help.Id)(implicit val ctx:
     path.segments.init.foldLeft(base.rawUrl / base.user.value / base.project.value / "main")(
       (uri, segment) => uri / segment.encoded
     )
-  private val url            = (rootUrl / "main").withPath(path)
+  private val mainUrl        = rootUrl / "main"
+  private val url            = (rootUrl / "main").withPath(mainUrl.path.concat(path))
```
Comment (rpiaggio) [inline explore/src/main/scala/explore/HelpBody.scala:40]: It took me a bit to understand what's going on here (and in line 79). Maybe we can extract a tiny method , say `concatToUriPath(uri: Uri, path: Uri.Path): Uri = uri.withPath(uri.path.concat(path))` ?

**Review by rpiaggio — APPROVED** [review]
LGTM

File: explore/src/main/scala/explore/HelpBody.scala (hunk 1 above)
Comment (cquiroz) [reply explore/src/main/scala/explore/HelpBody.scala:79]: yeah, it is a bit weird

**Pull request merged**